tools/docker/tests/test_flows.sh

```diff
@@ -82,8 +82,34 @@ test_ap_config_renew() {
 }
 
 test_ap_config_bss_tear_down() {
-    err "ap_config_bss_tear_down not implemented yet."
-    return 0
+    status "test ap config bss tear down"
+    # Regression test: MAC address should be case insensitive
```

Comment:
To make it a real regression test, I think it's better to add it with one case and to remove it with a different case. So the simplest would be to only use the uppercase version in the second dev_set_config.